test.R

```diff
@@ -163,6 +164,12 @@ testGGPlot <- function(description, generated, expected, show_expected = TRUE,
                      get_reporter()$add_message(feedback)
                      get_reporter()$end_test("", "wrong")
                  }
+                 if (show_expected && file.exists(tf_expected)){
```

Comment:
This is the same condition, is it not? If `show_expected == TRUE` then the file will have been generated? So `file.exists(tf_expected)` will also be true?

Reply:
In normal conditions it will but when ggsave fails then this wont' be the case and at the moment the function does yet catch the situation where an object is passed that is not a ggplot, but I will add that test rn.

Reply:
I've added a test to test the input plot object of ggsave and deleted the file.exists()